testing-daml-models/testing-with-scenarios/0_intro.md

```diff
@@ -0,0 +1,15 @@
+Once you've written a DAML model, you want to be certain that it behaves the way you imagined. DAML
+scenarios are a good way to unit test your DAML model and make sure it stays correct in further
+iterations.
+
+This tutorial will teach you how to
+
+1. write DAML scenarios 
+1. add assertions to you scenarios
+1. trace through your scenarios and output debug information
+
+We'll take the DAML model of the social network of the [GettingStarted](https://katacoda.com/daml/courses/getting-started)
```

Comment:
```suggestion
We'll take the DAML model of the social network of the [GettingStarted](https://daml.com/learn/getting-started/)
```

All scenarios will be available at daml.com/learn. Bernhard updated the Katacoda instructions.